client/state/ui/route/path/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
```

Comment:
We don't need that comment block.

Reply:
Yes we do, the linter will complain otherwise...